packages/template/vite/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@electron-forge/template-vite",
+  "version": "6.0.1",
+  "description": "Vite template for Electron Forge, gets you started with Vite really quickly",
+  "repository": "https://github.com/electron/forge",
+  "author": "caoxiemeihao",
+  "license": "MIT",
+  "main": "dist/ViteTemplate.js",
+  "typings": "dist/ViteTemplate.d.ts",
+  "scripts": {
+    "test": "mocha --config ../../../.mocharc.js test/**/*_spec.ts"
+  },
+  "engines": {
+    "node": "^14.18.0 || >=16.0.0"
```

Comment:
issue: I would argue that every single package in this monorepo should have the same `engines.node` value. What's the reason that you've changed it here?

Reply:
https://github.com/vitejs/vite/blob/v3.0.0/packages/vite/CHANGELOG.md#300-beta9-2022-07-08
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/26263658/203187800-9e027d05-3e24-4a70-9467-c718c2f05219.png">

https://github.com/vitejs/vite/blob/main/packages/vite/package.json#L33-L35
Not sure this is right, it just aligns with VIte.

Reply:
I think it makes sense to align with the minimum version of Vite if this is required. 👍